src/client/driver/driver.js

```diff
@@ -248,17 +254,52 @@ export default class Driver extends serviceUtils.EventEmitter {
         const childWindowDriverLink = new ChildWindowDriverLink(e.window, e.windowId);
 
         this.childWindowDriverLinks.push(childWindowDriverLink);
+        this._ensureChildWindowClosedWatcher();
+    }
+
+    _ensureChildWindowClosedWatcher () {
+        if (this.checkChildWindowClosedIntervalId)
+            return;
+
+        this.checkChildWindowClosedIntervalId = nativeMethods.setInterval.call(window, () => {
+            for (const childWindowDriverLink of this.childWindowDriverLinks) {
+                if (childWindowDriverLink.driverWindow.closed) {
```

Comment:
we can put this condition in `.filter` function

Reply:
Agree. I will change this code.